docs/recipes/WritingTests.md

```diff
@@ -304,7 +304,7 @@ import Header from '../../components/Header'
 
 function setup() {
   const props = {
-    addTodo: expect.createSpy()
+    addTodo: jest.fn(() => {})
```

Comment:
@HeinrichFilter Played with your branch a bit, you might want to change your package.json scripts to 

```
"test": "cross-env BABEL_ENV=commonjs jest",
```
otherwise you wont use the `transform-es2015-modules-commonjs` plugin.

Also, make sure to clear Jest cache on first launch, e.g. 

```
npm run test -- --no-cache
```

Having just `jest.fn()` seems OK with this setup.